### PR TITLE
Fix imu sensor

### PR DESCRIPTION
**Pull request by mattpow35**
Don't forget to update the readme in the holodeck repo :)
This is related to issue BYU-PCCL/holodeck#437

### CODE REVIEW

File: Source/Holodeck/Sensors/Private/IMUSensor.cpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ void UIMUSensor::TickSensorComponent(float DeltaTime, ELevelTick TickType, FActo
 
 void UIMUSensor::CalculateAccelerationVector(float DeltaTime) {
 	VelocityThen = VelocityNow;
-	VelocityNow = Parent->GetPhysicsAngularVelocityInDegrees();
+	VelocityNow = Parent->GetPhysicsLinearVelocity();
```
Comment (vinhowe) [inline Source/Holodeck/Sensors/Private/IMUSensor.cpp:50]: This looks right. On first glance it doesn't look like we have any tests that handle this.

File: enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// MIT License (c) 2019 BYU PCCL see LICENSE file
```
Comment (vinhowe) [inline enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp]: This looks like an auto-save file left behind by Visual Studio. Can you remove it?

**Review by vinhowe — CHANGES_REQUESTED** [review]
Will you revert your changes to `ExampleLevel.umap` and `ExampleLevel_BuiltData.uasset`? I think the fastest way is just `git checkout develop <path/to/ExampleLevel.umap> <path/to/ExampleLevel_BuiltData.uasset>`.

I'll ask @daniekpo but I don't think we have any tests that specifically handle the IMU sensor (I just looked), which means we don't have any tests to break but I guess we haven't specified any correct behavior either.

**Comment by daniekpo** [comment]
> Will you revert your changes to `ExampleLevel.umap` and `ExampleLevel_BuiltData.uasset`? I think the fastest way is just `git checkout develop <path/to/ExampleLevel.umap> <path/to/ExampleLevel_BuiltData.uasset>`.
> 
> I'll ask @daniekpo but I don't think we have any tests that specifically handle the IMU sensor (I just looked), which means we don't have any tests to break but I guess we haven't specified any correct behavior either.

Yeah, I don't think we have a test for the IMU sensor.

**Comment by daniekpo** [comment]
And, I'd revert the changes made to the asset files as @vinhowe recommended. I don't think you meant to make those changes.

File: enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp (hunk 2 above)
Comment (mattpow35) [reply enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp]: Good catch, I removed it.

**Comment by mattpow35** [comment]
I also did not see any tests for this, it would be a good idea to make some to make sure it works.

File: enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp [hunk 3]
```diff
@@ -0,0 +1,78 @@
+#include "Holodeck.h"
```
Comment (vinhowe) [inline enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp]: Can you get rid of this entire file? It looks like it's a autosaved copy of the file we're interested in.

Comment (mattpow35) [reply enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp]: Ya I will, my bad, I thought the last comment was only talking about the first line. 
Should I delete the entire directory named enc_temp_folder? this file seems to be the only thing in there

Comment (vinhowe) [reply enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp]: I went ahead and pushed https://github.com/BYU-PCCL/holodeck-engine/pull/219/commits/372471ac4020cd6d4eb8f5e3e5af75b20f1f35ac, which should take care of it.

Comment (mattpow35) [reply enc_temp_folder/347cdf8dc515add4ee3f91e33ca8de5/IMUSensor.cpp]: sweet

**Review by vinhowe — APPROVED** [review]
This looks good to me. @daniekpo do you want to block this until we have tests on the client? Seems to me like we could just go ahead and merge.

**Comment by daniekpo** [comment]
> This looks good to me. @daniekpo do you want to block this until we have tests on the client? Seems to me like we could just go ahead and merge.

I think we can just merge it.

**Pull request merged**